js/widgets/status.js

```diff
@@ -120,7 +120,8 @@ class StatusMatrix {
             const row = header.insertRow();
 
             cell = row.insertCell(); // i + 1);
-            cell.style.fontSize = Math.floor(this._cellScale * StatusMatrix.FONTSCALEFACTOR)*0.90 + "%";
+            cell.style.fontSize =
+                Math.floor(this._cellScale * StatusMatrix.FONTSCALEFACTOR) * 0.9 + "%";
```

Comment:
can you please explain what you've done here?

Reply:
As I mentioned in your previous review, this is just prettifying the code (as mentioned in #2609) and removing debug statement.